frontend/src/entry/overview/Overview.tsx

```diff
@@ -13,19 +14,25 @@ type OverviewPageKeys = (typeof OverviewPage)[keyof typeof OverviewPage]
 
 type OverviewProps = {
   entry: EntryInterface
+  readOnly?: boolean
   currentUserRoles: string[]
 }
 
-export default function Overview({ entry, currentUserRoles }: OverviewProps) {
+export default function Overview({ entry, currentUserRoles, readOnly = false }: OverviewProps) {
   const page: OverviewPageKeys = useMemo(
     () => (entry.card && entry.card.schemaId ? OverviewPage.FORM : OverviewPage.TEMPLATE),
     [entry.card],
   )
 
-  return (
+  return readOnly && !entry.card ? (
+    <MessageAlert
+      severity='warning'
+      message='This mirrored model has no model card. Please export the model card from the source model.'
+    />
+  ) : (
     <Container sx={{ my: 2 }}>
       {page === OverviewPage.TEMPLATE && <TemplatePage entry={entry} />}
-      {page === OverviewPage.FORM && <FormEditPage entry={entry} currentUserRoles={currentUserRoles} />}
+      {page === OverviewPage.FORM && <FormEditPage entry={entry} currentUserRoles={currentUserRoles} readOnly />}
```

Comment:
Whoops, just noticed this one. Passing `readOnly` without a specified value in react is the same as doing `readOnly={true}` so you'll need to update this to the following:
```
{page === OverviewPage.FORM && <FormEditPage entry={entry} currentUserRoles={currentUserRoles} readOnly={readOnly} />}
```